test/e2e/feature/feature.go

```diff
@@ -328,6 +328,9 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	UserNamespacesSupport = framework.WithFeature(framework.ValidFeatures.Add("UserNamespacesSupport"))
 
+	// TODO: document the feature (owning SIG, when to use this feature for a test)
```

Comment:
Please add documentation together the new feature.

@SergeyKanzhelev: would you have pointed out the same thing if I hadn't gotten involved?

I'm not sure how to ensure that developers do this without being reminded. Suggestions welcome :thinking:

Reply:
> @SergeyKanzhelev: would you have pointed out the same thing if I hadn't gotten involved?

Yes, after the PR with `TODO:`s was merged.